.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Packages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
```
Since we don't have changesets or any way of bumping the version number we should only publish this manually for now until we add this functionality